public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -42,30 +35,25 @@ import {
 import {
   LokiQuery,
   LokiOptions,
-  LokiLegacyQueryRequest,
-  LokiLegacyStreamResponse,
   LokiResponse,
   LokiResultType,
   LokiRangeQueryRequest,
   LokiStreamResponse,
 } from './types';
-import { LegacyTarget, LiveStreams } from './live_streams';
+import { LiveStreams, LegacyTarget } from './live_streams';
```

Comment:
Should probably rename `LegacyTarget` to make clear what it does (in relation to live targets).

Reply:
good point, thanks